src/controllers/attachment/index.ts

```diff
@@ -102,11 +103,14 @@ export class attachment extends Controller {
 
     if (!req.body && !file) throw new BadRequest('nothing to upload')
 
-    const [{ id, filename, binary_blob, created_at }]: any[] = await this.db
+    const buffer = Buffer.from(file?.buffer || JSON.stringify(req.body))
+    if (buffer.length > env.FILE_UPLOAD_MAX_SIZE) throw new BadRequest(`Over size limit: ${env.FILE_UPLOAD_MAX_SIZE}`)
```

Comment:
not immediately seeing how to set multer config when it's part of `tsoa`. I'll remove setting a size limit from this PR since it's not really part of what I'm doing + add a new ticket

Reply:
https://digicatapult.atlassian.net/browse/L3-117